test/test_cromwell_workflow.sh

```diff
@@ -9,38 +9,48 @@ INPUTS_JSON=$4
 WDL_FILE=$5
 OPTIONS_FILE=$6
 DEPENDENCIES_JSON=$7
+# Read list of dependency files into an array to pass to cromwell-tools
+DEPENDENCIES_LIST=$(cat ${DEPENDENCIES_JSON} | python3 -c "import json,sys;obj=json.load(sys.stdin);print(' '.join(obj.values()));")
```

Comment:
If you have `jq`, this could be a bit simpler, I think

Reply:
I was thinking that as well, but `jq` is not currently a part of the cromwell-tools docker image and I thought it would be confusing to add a dependency to cromwell-tools that it's not using. What do you think?

Reply:
Oh also I think this script would be much easier to read in python than in bash and we wouldn't have to rely on parsing JSON files. But that seemed out of scope for this ticket...so I'll leave it like this for now and we can improve it in the future!